### ISSUE TITLE
Request for original CMS-Template file

### ISSUE THREAD

**Issue by rawteech** [siuangie91/oop-php-photo-gallery#1]
Hi, could you please share with me the original template file... I will be so greatful

**Comment by siuangie91** [comment]
Hi, rawteech, all the files are available for download in lecture 5 of the course.

**Comment by rawteech** [comment]
@siuangie91 Yes, but I cant access them... I dont have a payment plan. Please assist me if you can

**Comment by Rubaabali** [comment]
it will be a request if you upload course files here? :)
